src/elf.rs

```diff
@@ -157,6 +158,7 @@ impl SectionBuilder {
             typ: SectionType::None,
             exec: false,
             write: false,
+            alloc: true,
```

Comment:
I wonder if it’s better to default this to false and explicitly call alloc on sections, but then again most sections are alloc I think? it’s not too important right now though